src/Main/main.css

```diff
@@ -0,0 +1,72 @@
+.container {
```

Comment:
you need to name your css files as water.css for example and put it to water folder